<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件批量重命名工具</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #f5f5f5;
}
QGroupBox {
    border: 1px solid #ccc;
    border-radius: 4px;
    margin-top: 10px;
    padding-top: 15px;
}
QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px;
}
QTreeView, QTableView, QTextEdit {
    border: 1px solid #ddd;
    border-radius: 3px;
    background: white;
}
QPushButton {
    min-width: 80px;
    padding: 5px 10px;
    border-radius: 4px;
}
QPushButton:hover {
    background: #e0e0e0;
}
QStatusBar {
    background: #e9e9e9;
    border-top: 1px solid #d4d4d4;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_Main">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_Path">
      <item>
       <widget class="QComboBox" name="comboBox_Path">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {
    padding: 3px;
    border-radius: 3px;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Refresh">
        <property name="text">
         <string>刷新</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_Path">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>6</number>
      </property>
      <widget class="QTreeView" name="treeView">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_Rules">
        <attribute name="title">
         <string>重命名规则</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_Rules">
         <item>
          <widget class="QGroupBox" name="groupBox_Replace">
           <property name="title">
            <string>替换规则</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_Replace">
            <item>
             <widget class="QCheckBox" name="checkBox_Replace">
              <property name="text">
               <string>启用替换</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_Find">
              <item>
               <widget class="QLabel" name="label_Find">
                <property name="text">
                 <string>查找内容：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_Find"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_Replace">
              <item>
               <widget class="QLabel" name="label_Replace">
                <property name="text">
                 <string>替换为：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_Replace"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_Regex">
              <property name="text">
               <string>使用正则表达式</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_Sequence">
           <property name="title">
            <string>序号规则</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_Sequence">
            <item>
             <widget class="QCheckBox" name="checkBox_Sequence">
              <property name="text">
               <string>启用序号</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_Sequence">
              <item>
               <widget class="QLabel" name="label_Start">
                <property name="text">
                 <string>起始值：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_Start">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>9999</number>
                </property>
                <property name="value">
                 <number>1</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_Step">
                <property name="text">
                 <string>步长：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_Step">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>1</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_Digits">
              <item>
               <widget class="QLabel" name="label_Digits">
                <property name="text">
                 <string>最小位数：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_Digits">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>10</number>
                </property>
                <property name="value">
                 <number>3</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_Position">
                <property name="text">
                 <string>位置：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_Position">
                <item>
                 <property name="text">
                  <string>前缀</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>后缀</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>替换原名</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_Other">
           <property name="title">
            <string>其他规则</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_Other">
            <item>
             <widget class="QCheckBox" name="checkBox_Case">
              <property name="text">
               <string>修改大小写</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_Case">
              <item>
               <property name="text">
                <string>全部大写</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>全部小写</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>首字母大写</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_Extension">
              <property name="text">
               <string>保留扩展名</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_Buttons">
           <item>
            <widget class="QPushButton" name="pushButton_Preview">
             <property name="text">
              <string>生成预览</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/icons/preview.png</normaloff>:/icons/preview.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_Execute">
             <property name="text">
              <string>执行重命名</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/icons/execute.png</normaloff>:/icons/execute.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_Buttons">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_Preview">
        <attribute name="title">
         <string>预览结果</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_Preview">
         <item>
          <widget class="QTableView" name="previewTable">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_Log">
        <attribute name="title">
         <string>操作日志</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_Log">
         <item>
          <widget class="QTextEdit" name="textEdit_Log">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>647</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_OpenFolder"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_OpenFolder"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_OpenFolder">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/folder_open.png</normaloff>:/icons/folder_open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件夹</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="separator">
   <property name="separator" stdset="0">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_OpenFolder</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onOpenFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Preview</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onGeneratePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Execute</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onExecuteRename()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_Path</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>onPathComboChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Refresh</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>fsModel-&gt;setRootPath(fsModel-&gt;rootPath())</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
